common/laws/00_education_system.txt

```diff
@@ -38,13 +38,26 @@ law_religious_schools = {
 		law_state_atheism
 		law_serfdom
 	}
+
 	institution = institution_schools
 	institution_modifier = {
-		state_conversion_mult = 0.2
-		state_education_access_add = 0.1
-		interest_group_ig_devout_pol_str_mult = 0.1
+		state_conversion_mult = 0.25
+		state_assimilation_mult = 0.05
+		state_education_access_add = 0.07
+		interest_group_ig_devout_pol_str_mult = 0.05
+		interest_group_ig_landowners_pol_str_mult = 0.03
+		interest_group_ig_devout_pop_attraction_mult = 0.05
+		country_loyalists_from_legitimacy_mult = 0.03
+		country_radicals_from_legitimacy_mult = -0.02
+		state_accepted_birth_rate_mult = 0.01
+		state_pop_support_law_state_religion_add = 0.10
+		state_pop_support_law_denominational_exclusion_add = 0.05
+		state_pop_support_law_freedom_of_conscience_add = 0.05
+		state_pop_support_law_total_separation_add = -0.1
+		state_pop_support_law_state_atheism_add = -0.15
```

Comment:
These are flat adds, so regardless of the amount of pops actually support it, it gets added. I think you meant relative multipliers?